src/tcgwars/logic/impl/gen8/ChillingReign.groovy

```diff
@@ -2707,14 +2709,14 @@ public enum ChillingReign implements LogicCardInfo {
         return evolution (this, from:"Metagross V", hp:HP330, type:M, retreatCost:3) {
           weakness FIRE
           resistance GRASS, MINUS30
-          move "Magnetic Pull", {
+          move "Zap Traction", {
             text "Search your deck for up to 2 cards and put them into your hand. Then, shuffle your deck."
             energyCost METAL
             attackRequirement {
               assert my.deck : "Your deck is empty"
             }
             onAttack{
-              my.deck.select(max: 2).moveTo(hidden: true, my.hand)
+              my.deck.select(min: 0, max: 2).moveTo(hidden: true, my.hand)
```

Comment:
Attacks that "search for up to x _cards_" must take at least one, when a certain filter isn't specified

```suggestion
              my.deck.select(min: 1, max: 2).moveTo(hidden: true, my.hand)
```